ui/src/main/resources/PhenoTips/VariantDisplay.xml

```diff
@@ -366,8 +366,8 @@
     		if (r.chrom &amp;&amp; r.position &amp;&amp; r.ref) {
 	    		var start = r.position;
 	    		var end = r.position + r.ref.length;
-			var positionText = _this.Strings["positionPhrase"].replace("__CHROMOSOME__",r.chrom.toUpperCase()).replace("__START__", start).replace("__STOP__", end);
-	    		var positionURLFragment = "Chr" + r.chrom.toUpperCase()[0] + ": " + start + "-" + end;
+	    		var positionText = _this.Strings["positionPhrase"].replace("__CHROMOSOME__",r.chrom.toUpperCase()).replace("__START__", start).replace("__STOP__", end);
```

Comment:
This spacing made me sad. Looks like the whole file uses a weird combination of tabs and spaces, with these lines having the format: <TAB><4 spaces><TAB><TAB><TAB>...
I decided to be consistent.